Unaccountable-Magpie/BangazonSite
# Search bar

### Purpose:

Allows users to search for products

### How it fits in context of the project:
(Link to ticket: # )

fulfills ticket #6 

### Specific Feature affected:

Layout.cs
productsController.cs


### How to test (Be thorough!):

1. Check into my branch bs-search
2. Run the build
3. On the home page there should be a search bar
4. Search for anything like "socks"
5. This should route you to the product index page and display the search results
6. Run Dotnet Test in the terminal
7. There should be 4 tests and 0 failures
